Dockerfile

```diff
@@ -77,9 +77,17 @@ COPY presto-raptor-legacy /build/presto-raptor-legacy
 COPY presto-password-authenticators /build/presto-password-authenticators
 COPY src /build/src
 COPY pom.xml /build/pom.xml
+#build debug info
+RUN cat /proc/meminfo
+
+# increase memory available to Maven
+RUN export MAVEN_OPTS='-Xmx2048m -XX:MaxPermSize=128m'
```

Comment:
Is there any reason we need to export these maven options, or can we use `ENV` instead? iirc `RUN export ...` won't persist through intermediate images.